client/src/components/App.jsx

```diff
@@ -1,14 +1,47 @@
 import React from 'react';
+import Details from './Details.jsx';
 
 class App extends React.Component {
-  constructor() {
-    super();
-    this.state = {};
+  constructor(props) {
+    super(props);
+    this.state = {
+      hostName: '',
+      city: '',
+      country: '',
+      gettingAround: '',
+      description: '',
+      guidebookId: null
+    };
   }
 
+  componentDidMount() {
+
+    // $.ajax({
+    //   url: '/api/rooms/11111111/neighborhood',
+    //   dataType: 'json',
+    //   method: 'GET',
+    //   contentType: 'application/json',
+    //   // data: JSON.stringify({jsonData: data}),
+    //   success: function(data) {
+    //     console.log(data);
+    //   },
+    //   error: function(error) {
+    //     console.log(error);
+    //   },
+    //
+    // });
+
+  };
+
   render() {
     return (
-      <div className="helloworld">Hello World!</div>
+      <div className="App">
+        <h2>The neighborhood</h2>
+        <p className="description">{this.state.name}'s place is located in {this.state.city}, {this.state.country}.</p>
+        <p>{this.state.description}</p>
+        <p>{this.state.gettingAround}</p>
```

Comment:
Putting this information in a 'p' tag will come along with some extra styles and padding associated with the <p> tag. I see you're giving it a class - maybe just make sure you're not getting any additional styles that come along with the <p> tag that you don't want.